src/main/java/net/sf/log4jdbc/StatementSpy.java

```diff
@@ -34,6 +34,20 @@
  */
 public class StatementSpy implements Statement, Spy
 {
+
+  static final boolean showParams;
+
+  static {
+    final String f = Boolean.FALSE.toString();
+    final String key = "LOG4JDBC_SQL_SHOW_PARAMS";

```

Comment:
Like Hasher, we should check system property then environment variable for `LOG...PARAMS`. Are we doing that if not here?

Reply:
I do not remember we do this in Hasher, but it's not a problem to add if needed.

Reply:
I added checking both system property and env property, please take a look.